src/dnd/DragArea.jsx

```diff
@@ -10,13 +11,15 @@ class DragArea extends Component {
     lastY: 0,
     deg: 0,
     draggableId: null,
-    droppableId: null,
+    lastDroppableId: null,
     droppableContainers: [],
+    containers: this.props.containers,
   };
 
   draggable = null;
 
   setDraggable = (draggable, draggableId) => {
+    // TODO: resolve the problem with finding z-index
```

Comment:
Let's resolve right now with help of not used `classnames`

This PR should be concluding regarding base dnd UI